docs/modders/Entities_Format/Creature_Help.md

```diff
@@ -0,0 +1,154 @@
+
+Make animations seamless from one to another. To do this, you can draw the first and the last images with a prompt with _ToonCrafter_: https://huggingface.co/spaces/ChristianHappy/tooncrafter
+
+Most of the time, you need to increase the resolution or the quality of your template image, so use _SUPIR_: https://huggingface.co/spaces/Fabrice-TIERCELIN/SUPIR
+
+## Battle sound effect
+
+To create the audio effects, I recommend to use _Tango 2_: https://huggingface.co/spaces/declare-lab/tango2
+
+The quality is better than _Stable Audio_.
+
+## Map render
+
+We don't know the right elevation angle for the view but 45° elevation seems to be a good choice. For the sunlight direction, I would say 45° elevation and 45° azimut.
+
+The map creatures are not rendered on the map with vanishing points but in isometric. You can [get an orthogonal render in Blender](https://blender.stackexchange.com/a/135384/2768). If you are creating a creature and its updated version, most of the time, the both creatures are not oriented to the same side on the map. I think that the animation on the map is usually the _Mouse Over_ animation on battle.
+
+You can see that the view angle is higher than on a battle. To change the angle from a battle sprite, you can use _Zero 1-to-3_: https://huggingface.co/spaces/cvlab/zero123-live
+
+You can get higher resolution using this Video AI that can control the motion of the camera: https://huggingface.co/spaces/TencentARC/MotionCtrl_SVD
+
+If you have a 3D software, you can get better quality by converting your image into 3D model and then render it from another angle using _Stable Fast 3D_: https://huggingface.co/spaces/stabilityai/stable-fast-3d
+
+Follow this comment to retrieve the color: https://huggingface.co/stabilityai/TripoSR/discussions/1#65e8a8e5e214f37d85dad366
+
+### Shadow render
+
+There are no strong rules in the original game about the angle of the shadows on the map. Different buildings have inconsistent shadows. To draw the shadow, I recommend the following technique:
+
+Let's consider that the object is a vertical cone:
+| | | | | | | | | | |
+|---|---|---|---|---|---|---|---|---|---|
+| c | c | c | c | c | c | c | c | c | c |
+| c | c | c | c | c | c | c | b | c | c |
+| c | c | c | c | c | c | c | b | c | c |
+| c | c | c | c | c | c | b | b | b | c |
+| c | c | c | c | c | c | b | b | b | c |
+| c | c | c | c | c | b | b | b | b | b |
+| c | c | c | c | c | c | b | b | b | c |
+| c | c | c | c | c | c | c | c | c | c |
+| c | c | c | c | c | c | c | c | c | c |
+| c | c | c | c | c | c | c | c | c | c |
```

Comment:
perhaps remove 'c''s or replace them with single dot to make this table more easy to read?